libcontainer/seccomp/seccomp_linux.go

```diff
@@ -86,6 +86,29 @@ func InitSeccomp(config *configs.Seccomp) (int, error) {
 		}
 	}
 
+	// Add extra flags
+	for _, flag := range config.Flags {
+		switch flag {
+		case "SECCOMP_FILTER_FLAG_TSYNC":
```

Comment:
Note: I think we should have them as consts in the runtime spec. I proposed that change in https://github.com/opencontainers/runtime-spec/pull/1138

Reply:
Note that seccomp-golang sets this flag unconditionally, and we (runc/libcontainer/seccomp/patchbpf) do not set this at all (not sure if this was by design or just forgotten; @cyphar WDYT?)

From what I see, it seems that for runc it does not make sense to do so, as we execute runc init right after applying seccomp rules, and it does not matter much whether these rules were applied to all threads or not.

Reply:
There was an issue with setting it in patchbpf, I can't remember the exact details.